provider/datagen/src/baked_exporter.rs

```diff
@@ -558,15 +593,58 @@ impl DataExporter for BakedExporter {
                                                 payload: Some(payload),
                                             }
                                         })
-                                        .ok_or_else(|| icu_provider::DataErrorKind::MissingLocale.with_req(<#markers as icu_provider::KeyedDataMarker>::KEY, req))
+                                        .ok_or_else(|| icu_provider::DataErrorKind::MissingLocale.with_req(<#marker as icu_provider::KeyedDataMarker>::KEY, req))
                                     }
                             }
                         }
                     }
+                }
+            )?;
+        }
+
+        self.write_to_file(
+            PathBuf::from("macros"),
+            quote! {
+
+                #[macro_use]
+                mod macros {
+                    #(
+                        #[macro_use]
+                        mod #mod_idents;
+                    )*
+                }
```

Comment:
Maybe `include!` the files instead?. I'd like to avoid macro scoping headaches as much as possible.

Reply:
Nested includes are also sketchy, at least macro exporting is well documented (and the reexport trick is super common)